packages/obonode/obojobo-chunks-code/schema.js

```diff
@@ -26,13 +26,13 @@ const schema = {
 
 						return editor.wrapBlockByKey(child.key, {
 							type: CODE_LINE_NODE,
-							data: { content: { indent: 0 } }
+							data: { content: { indent: 0, hangingIndent: false } }
```

Comment:
Because code lines never wrap, I don't think that we need hanging indent values on them in the editor